### ISSUE TITLE
Keybase client on macOS displaying an error when I click on "chats"

### ISSUE THREAD

**Issue by shoenseiwaso** [keybase/client#23828]
I opened Keybase today and was greeted with a screenshot similar to the below. I submitted my logs with `keybase log send`, restarted Keybase, and then saw this same error screen when i clicked on "chats"

my log id: ce0cd23012d63402c9c9bc1c

![image](https://user-images.githubusercontent.com/5861379/79524887-93bb4a80-809c-11ea-927b-74ee7ea64d04.png)

**Comment by mmaxim** [comment]
@shoenseiwaso Try completely quitting Keybase (from the menu bar) and restarting it, it seems like you have a version mismatch between parts of the app.

**Comment by shoenseiwaso** [comment]
@mmaxim I did! That screenshot is from after I "Keybase > Quit Keybase completely" and restarted.

**Comment by jeffw-wherethebitsroam** [comment]
I had to manually kill off a keybase service. First I quit keybase, then in the terminal I ran:

```
% ps aux | grep -i keybase
jeffwilliams      2859   8.6 16.5  7637520 2763268   ??  S     6Apr20  18:39.94 /Applications/Keybase.app/Contents/SharedSupport/bin/keybase -d --log-file=/Users/jeffwilliams/Library/Logs/keybase.service.log service

% kill 2859
```

After this, I started Keybase again and everything is happy. So it seems the upgrade left an old service running?

**Comment by mmaxim** [comment]
Yeah my next suggestion was going to be to reboot to effectively do what you
just did. We are trying to track this down, for some reason an old service can get stuck here and be out of sync with the new UI. Sorry for the trouble, and glad you are back in business.
